src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
The class 'Circle' should not start with an empty line. Please remove the empty line at the beginning of the class definition.